buildscripts/condarecipe.local/meta.yaml

```diff
@@ -27,6 +28,7 @@ requirements:
     - llvmlite 0.25.*
     - funcsigs                 # [py27]
     - singledispatch           # [py27]
+    - tbb-devel                # [not (armv6l or armv7l or aarch64)]
```

Comment:
TBB should actually work there, is it just not ready on anaconda channel?

Reply:
I think so, fundamentally it's just C++ so there's a reasonable chance of it compiling fine, just hasn't been tried yet.